src/components/FilterSideBar/FilterSideBar.js

```diff
@@ -87,7 +95,16 @@ const FilterSideBar = () => {
 
         <div className="sidebar-filters">
           <h3>Price</h3>
-          <input type="range" />
+          <label htmlFor="priceSlider">$0 to ${rangeValue}</label>
+          <input
+            type="range"
+            min="0"
+            max="1000"
+            value={rangeValue}
+            onChange={(e) =>
+              filterDispatch({ type: "PRICE_FILTER", payload: e.target.value })
```

Comment:
use `htmlFor` for label and `id` for input elements for all the labels and inputs. This helps in accessibility